LDM-592.tex

```diff
@@ -171,15 +171,15 @@ \section{Alert Production}
 The user, a Staff Scientist, specifies a visit to be processed.
 
 \item
-The butler expands the incomplete DatasetExpresion by setting the filter and selecting only those CCDs for which raw image data is available.
+The butler expands the DatasetExpresion by setting the filter and selecting only those CCDs for which raw image data is available.
```

Comment:
I'd prefer: "The butler expands the DatasetExpresion to select only those CCDs for which raw image data is available."